optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -118,42 +117,34 @@ Each of these families of algorithms has multiple optimization algorithms:
 |    <<geneticAlgorithms,Genetic Algorithms>> |4/5 |3/5 |2/5 |5/5|Yes
 |===
 
-If you want to learn more about metaheuristics, read the free books http://www.cs.gmu.edu/~sean/book/metaheuristics/[Essentials of Metaheuristics] or http://www.cleveralgorithms.com/[Clever Algorithms].
+To learn more about metaheuristics, see  http://www.cs.gmu.edu/~sean/book/metaheuristics/[Essentials of Metaheuristics] or http://www.cleveralgorithms.com/[Clever Algorithms].
 
 
 [[whichOptimizationAlgorithmsShouldIUse]]
 == Which Optimization Algorithms Should I Use?
 
-The _best_ optimization algorithms configuration for your use case depends heavily on your use case.
-Nevertheless, this vanilla recipe will get you into the game with a pretty good configuration, probably much better than what you're used to.
+The best optimization algorithms configuration to use depends heavily on your use case.
+Nevertheless, this basic procedure will provide basic configuration.
```

Comment:
"This basic procedure provides a good starting configuration that will produce better than average results."?